modules/move-etcd-different-disk.adoc

```diff
@@ -180,11 +181,31 @@ spec:
         enabled: true
         name: var-lib-etcd.mount
 ----
-. Apply the modified version that removes the logic for creating and syncing the device by entering the following command:
+
+. Apply the modified version that removes the logic for creating and syncing the device to prevent the nodes from rebooting:
```

Comment:
```suggestion
. Apply the modified version that removes the logic for creating and syncing the device to prevent the nodes from rebooting. Run the following command:
```